### ISSUE TITLE
Potential addition of Mapping Quality filter parameter to initial mapping step

### ISSUE THREAD

**Issue by DarianHole** [polio-nanopore/piranha#110]
Hi!

I'm curious if it would be useful to add a mapping quality filter for the initial mapping step? We've seen a few cases where we are getting Coxsackievirus (A20 I believe) reads mapping relatively weakly to WPV1 (15 MQ) VP1 region. I'm not super familiar with all of the enteroviruses and their similarities so not sure if this would be helpful.

Any thoughts?

Darian

**Comment by DarianHole** [comment]
Yeah no worries on the timeline! Piranha has been working great for us so far as is!! 

We have some of our own pre and post processing steps to double check/confirm anything we see that is suspicious. I also still need to look at optimizing the resources I have available

**Comment by aineniamh** [comment]
Thanks @DarianHole, glad to hear it! Also very happy to take more suggestions, or if there's anything suspicious that tends to crop up we could put in some automated checks for if its helpful!

**Comment by aineniamh** [comment]
Closing as I believe this is functional now!
